pycbc/inference/models/tools.py

```diff
@@ -421,7 +421,7 @@ def draw_sky_times(self, snrs):
             end = min(tcmax + EARTH_RADIUS, snr.end_time)
             snr = snr.time_slice(start, end, mode='nearest')
 
-            w = snr.squared_norm().numpy()
+            w = snr.squared_norm().numpy() / 2.0
```

Comment:
Wait, is that a bug? Should the divide by 2 have been there before? If not, does this break anything?

Reply:
It should have been a 2, but it didn't affect the testing so far or results. It previously would only affect the choice of monte-carlo samples to be slightly less efficient than otherwise. The weights were self-consistent otherwise, hence why no observed issue in the results themselves before.